src/hexo/themes/documentation/layout/partials/sub-page.hbs

```diff
@@ -21,6 +21,16 @@
     </nav>
     <header class="page-intro">
         <h1 id="page-heading">{{page.title}}</h1>
+        <div class="documentation-actions">
+            <ul>
+                <li>
+                    <a target="_blank" href="{{theme.sonarRepoRoot}}/blob/master{{getMarkdownLink page.permalink}}">Edit</a>
+                </li>
+                <li>
+                    <a target="_blank" href="{{theme.sonarRepoRoot}}/issues/new?title=[docs]%20Issue%20with%20%27%3C{{page.title}}%3E%27 ">Signal Issue</a>
```

Comment:
@qzhou1607 Maybe drop the `<`, `>`?

Also, should we add a link to the original URL in the issue page?